wiki/iNiS_games/id.md

```diff
@@ -14,13 +14,13 @@ tags:
   - Ouendan
 ---
 
-# Gim dari iNiS
+# Permainan-permainan dari iNiS
```

Comment:
ehh i'm still not sure about this one tho... although `Permainan-permainan dari iNiS` ("Games from iNiS") is *technically* correct i feel like `Permainan-permainan iNiS` ("iNiS Games") would infinitely be more fitting here as the article's title ;~;

Reply:
wait, I'm drunk I forgot to remove `dari`

Reply:
yeah it's all good dw lol